helm-chart/eoapi/values.yaml

```diff
@@ -346,6 +346,7 @@ stac:
       PORT: "8080"
       # https://www.uvicorn.org/settings/#production
       WEB_CONCURRENCY: "5"
```

Comment:
🤔 if we add WEB_CONCURRENCY set to 5 and then min_conn_size set to 10 (the default in stac-fastapi-pgstac) did it meant we were trying to have 5 * 10 = 50 connections 😬

Reply:
well in fact there are 2 pools openned at startup (read/write) so it meant 100 connections 😱